bin/tessel-2.js

```diff
@@ -196,6 +196,29 @@ parser.command('rename')
   .option('timeout', timeoutOption)
   .help('Change the name of a Tessel to something new.');
 
+parser.command('update')
+  .option('build', {
```

Comment:
`build` -> `version` because that's the name of the property being read for this operation and that's the word used in the `tessel update list` output